google_guest_agent/main.go

```diff
@@ -89,12 +89,14 @@ func closeFile(c io.Closer) {
 func availableManagers() []manager {
 	if runtime.GOOS == "windows" {
 		return []manager{
+			&addressMgr{},
```

Comment:
since we have a common manager, do you want to keep a list of common managers and append to it based on os type?

Reply:
Could be, I can change it.

Reply:
done. thanks.